### DISCUSSION TITLE
I need help for my project

### DISCUSSION THREAD

**Discussion by kayahuseyin** [leolelego/brickie.swiftui#115] [Q&A]
Hi,

Thank you for this great project. I'm currently working on a small app that displays Lego sets based on the user's theme selection. However, I'm encountering difficulties in geting data from the API. I've reviewed your NetworkCalls, but unfortunately, I couldn't figure it out. It's only been a few months since I started iOS Development. Could you please help me with this? I would really appreciate it.

**Comment by leolelego** [comment]
Hi, learn how to read a documentation on Brickset.com
